packages/apps-config/src/api/spec/index.ts

```diff
@@ -99,13 +97,11 @@ import zero from './zero';
 
 // NOTE: The mapping is done from specName in state.getRuntimeVersion
 const spec: Record<string, OverrideBundleDefinition> = {
-  Crab: crab,
-  Darwinia: darwinia,
+  ...darwinia,
```

Comment:
I really would prefer an explicit mapping here - it is indeed more of a pain, however it also ensures that no other keys are overridden unintentionally. You can still import the bundle from the single file, i.e.

```js
import { Crab, Darwinia } from './darwinia';

...
{
...
  Crab,
  Darwinia,
...
}
```

Reply:
Has changed explicit mapping.